commit 6543d651d30c617c991dd49a35b7531d377d7b06
Author: Mike Hiretsky <mh@calculate.ru>
Date:   Wed Aug 3 13:51:28 2011 +0400

    Fix detect network by ip and mask if ip and mask absent.

diff --git a/pym/cl_fill_install.py b/pym/cl_fill_install.py
index b8acbd9..a03b841 100644
--- a/pym/cl_fill_install.py
+++ b/pym/cl_fill_install.py
@@ -121,7 +121,10 @@ class fillVars(object, glob_attr):
             mapInterface = {}
             mapInterface["ip"] = ipaddr
             mapInterface["cidr"] = str(ip.maskToCidr(mask)) if mask else ""
-            mapInterface["network"] = ip.getIpNet(ipaddr,mask=mask)
+            if ipaddr and mask:
+                mapInterface["network"] = ip.getIpNet(ipaddr,mask=mask)
+            else:
+                mapInterface["network"] = ""
             mapInterface["dhcp"] = "on" if dhcp else "off"
             mapInterface["mac"] = mac
             mapInterface["name"] = "{vendor} {name}".format(**pciInfo)
